app/src/main/java/com/marekmacko/kotlinapp/util/Utils.kt

```diff
@@ -10,7 +10,7 @@ class Utils {
         fun isNetworkAvailable(context: Context): Boolean {
             val connectivityManager = context.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
             val networkInfo = connectivityManager.activeNetworkInfo
-            return networkInfo?.isConnected ?: false
+            return networkInfo?.isConnected == true
```

Comment:
Couldn't be just return?

Reply:
No, because there could be situation when `networkInfo` will be null and this will be inconsistent with return type.